services/gateway/apis/http-version3/swagger/definitions/tokens.json

```
@@ -142,6 +142,41 @@
             }
         }
     },
+    "escrowedAmount": {
+        "type": "object",
+        "required": [
+            "escrowChainID",
+            "tokenID",
+            "amount"
+        ],
+        "properties": {
+            "escrowChainID": {
+                "type": "string",
+                "example": "00000001",
+                "description": "Chain ID for the escrowed token"
```

Please make it a consistent habit of ending all the logs, summaries and descriptions with a `.`. Also, please make an attempt to ensure that all the examples in the documentation always corresponds to the Lisk Mainchain. Kindly rectify the same for any other instance within the swagger documentation.
```suggestion
               "example": "00000000",
               "description": "ID of the chain to which the specified token in escrowed."
```

---

applied.